### ISSUE TITLE
Some smaller tests can be slow..

### ISSUE THREAD

**Issue by ekluzek** [ESCOMP/ctsm#402]
With a small cime update I saw that the following tests were slow:

ERP_P36x2_D_Ld5.f10_f10_musgs.I2000Clm50Cn.cheyenne_gnu.clm-default.GC.fincln01dev012chgnua/TestStatus:FAIL ERP_P36x2_D_Ld5.f10_f10_musgs.I2000Clm50Cn.cheyenne_gnu.clm-default TPUTCOMP Error: Computation time increase > 25 pct from baseline

ERP_P36x2_Lm13.f10_f10_musgs.IHistClm50Bgc.cheyenne_gnu.clm-monthly.GC.fincln01dev012chgnua/TestStatus:FAIL ERP_P36x2_Lm13.f10_f10_musgs.IHistClm50Bgc.cheyenne_gnu.clm-monthly TPUTCOMP Error: Computation time increase > 25 pct from baseline

ERS_Ly20_Mmpi-serial.1x1_numaIA.I2000Clm50BgcDvCropQianGs.cheyenne_intel.clm-cropMonthOutput.GC.fincln01dev012chintela/TestStatus:FAIL ERS_Ly20_Mmpi-serial.1x1_numaIA.I2000Clm50BgcDvCropQianGs.cheyenne_intel.clm-cropMonthOutput TPUTCOMP Error: Computation time increase > 25 pct from baseline

ERS_Ly6_Mmpi-serial.1x1_smallvilleIA.IHistClm50BgcCropGs.cheyenne_intel.clm-cropMonthOutput.GC.fincln01dev012chintela/TestStatus:FAIL ERS_Ly6_Mmpi-serial.1x1_smallvilleIA.IHistClm50BgcCropGs.cheyenne_intel.clm-cropMonthOutput TPUTCOMP Error: Computation time increase > 25 pct from baseline

PET_P36x2_D.f10_f10_musgs.I1850Clm50BgcCrop.cheyenne_gnu.clm-default.GC.fincln01dev012chgnua/TestStatus:FAIL PET_P36x2_D.f10_f10_musgs.I1850Clm50BgcCrop.cheyenne_gnu.clm-default TPUTCOMP Error: Computation time increase > 25 pct from baseline

SMS_D_Ld5_Mmpi-serial.1x1_mexicocityMEX.I1PtClm50SpGs.cheyenne_intel.clm-default.GC.fincln01dev012chintela/TestStatus:FAIL SMS_D_Ld5_Mmpi-serial.1x1_mexicocityMEX.I1PtClm50SpGs.cheyenne_intel.clm-default TPUTCOMP Error: Computation time increase > 25 pct from baseline

SMS_D_Lm1_Mmpi-serial.CLM_USRDAT.I1PtClm50SpGs.cheyenne_intel.clm-USUMB.GC.fincln01dev012chintela/TestStatus:FAIL SMS_D_Lm1_Mmpi-serial.CLM_USRDAT.I1PtClm50SpGs.cheyenne_intel.clm-USUMB TPUTCOMP Error: Computation time increase > 25 pct from baseline

SMS_D_Ly2.1x1_brazil.IHistClm50BgcQianGs.cheyenne_intel.clm-ciso_bombspike1963.GC.fincln01dev012chintela/TestStatus:FAIL SMS_D_Ly2.1x1_brazil.IHistClm50BgcQianGs.cheyenne_intel.clm-ciso_bombspike1963 TPUTCOMP Error: Computation time increase > 25 pct from baseline

In talking this over with @fischer-ncar he said he's seen variability in smaller tests on cheyenne, so we want to mark this as expected. The single point cases probably especially can be variable because they are in shared queue. So we should mark these as allowing greater variability and make sure they have enough wallclock for that variability.

**Comment by billsacks** [comment]
@ekluzek it's not clear to me what you're asking for with this issue. I typically (okay, always) ignore TPUTCOMP failures because they can be so variable, and I'd suggest we continue to do so. I don't think we should bother adding these to the expected fails list: I feel this will be a lot of work without much benefit.

So can we close this as a wontfix?

**Comment by ekluzek** [comment]
@billsacks, the purpose is given in the last paragraph. I just want to make sure these tests are flagged as having greater variability, and that they have enough wall clock time even on a bad day for cheyenne. I'm not going to mark them as expected fails, but I am upping their tput_tolerance, and I upped the wallclock time for a few of them.

The TPUTCOMP is much better than it used to be, and for it to be useful ever, we do need to set the tput_tolerance for tests in our test list. I thought tput_tolerance was a valid option, but it turns out I had to add it into cime, but it was easy so I have a PR in cime for that. Once, it's in, the tput_tolerance settings in our test list will become active.
